Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+gem 'xmlrpc'
```

Comment:
For future reference @chrisarcand @bdunne ... This is required for Ruby 2.4 because it was ripped out of stdlib and into a gem.

Reply:
And it actually belongs in active_bugzilla.

Reply:
yup!